deployment/docker-compose/litellm_proxy_config.yaml

```diff
@@ -16,17 +16,20 @@ model_list:
     litellm_params:
       model: gpt-4o
       api_key: "os.environ/OPENAI_API_KEY"
-      # model: gemini/gemini-1.5-pro-latest
-      # api_key: "os.environ/GEMINI_API_KEY"
-      # safety_settings:
-      #   - category: HARM_CATEGORY_HARASSMENT
-      #     threshold: BLOCK_NONE
-      #   - category: HARM_CATEGORY_HATE_SPEECH
-      #     threshold: BLOCK_NONE
-      #   - category: HARM_CATEGORY_SEXUALLY_EXPLICIT
-      #     threshold: BLOCK_NONE
-      #   - category: HARM_CATEGORY_DANGEROUS_CONTENT
-      #     threshold: BLOCK_NONE
+  - model_name: generate-gemini-response
+    litellm_params:
+      # Set VERTEXAI_ENDPOINT environment variable or direclty enter the value:
```

Comment:
Where would you directly set this? In line 22?

Reply:
Right right..